vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -0,0 +1,57 @@
+# How to customize VizroAI
+
+This guide explains how to customize `VizroAI`.
+
+`VizroAI` accepts a single argument named `model`. This `model` argument can either be a string that specifies the name of the `ChatOpenAI` model or an instantiated `ChatOpenAI` model.
+
+We'll delve into customizing the `ChatOpenAI` model later; for now, let's concentrate on the default usage of `VizroAI`.
+
+## Default usage
+
+`VizroAI`can be initialized without any arguments, in which case it will, by default, use OpenAI model `"gpt-3.5-turbo"` with a temperature setting of 0.
```

Comment:
```suggestion
`VizroAI` can be initialized without any arguments, in which case it will, by default, use the OpenAI model `"gpt-3.5-turbo"` with a temperature setting of 0.
```